### PR TITLE
Added support for additional model args

### PR DESCRIPTION
**Pull request by JonahSussman**

### CODE REVIEW

File: kai/config.toml [hunk 1]
```diff
@@ -21,8 +21,13 @@ password = "dog8code"
 #   OpenAI GPT 4
 #     provider = "OpenAI"
 #     args = { model_id = "gpt-3.5-turbo" }
-provider = "IBMGranite"
-args = { model_id = "ibm/granite-13b-chat-v2" }
+# provider = "IBMGranite"
+# args = { model_id = "ibm/granite-13b-chat-v2" }
```
Comment (jwmatthews) [inline kai/config.toml:25]: Is the intent to not have a model configured by default?

Comment (JonahSussman) [reply kai/config.toml:25]: Mistype, fixing

**Review by jwmatthews — APPROVED** [review]
Should we also add in `max_new_tokens` and `min_new_tokens`?

**Comment by JonahSussman** [comment]
Sounds good, will add

**Pull request merged**